File: java/8.11/example8.java
import java.util.Scanner;

public class example8 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the samllest number");
        Integer firstNumber = Integer.parseInt(scanner.nextLine());
        System.out.println("Enter the biggest number");
        Integer secondNumber = Integer.parseInt(scanner.nextLine());
        outputNumbers(firstNumber, secondNumber);
    }

    public static void outputNumbers(Integer num1, Integer num2) {
        for (Integer count = num1; count <= num2; count++) {
            System.out.println(count);
        }
    }
}

File: 8.11/example8.py
def outputNumbers(num1, num2): # Creating a def command that requiers to variables
    for count in range(num1, num2+1): # Running a loop in bretween the to variables
        print(str(count)) # Printing

firstNumber = int(input("Enter the samllest number")) # Setting the value to the interger of the value inputed
secondNumber = int(input("Enter the largest number")) # Setting the value to the interger of the value inputed
outputNumbers(firstNumber, secondNumber) # Running outputNumbers with supplying the two variables